### ISSUE TITLE
folding using special characters

### ISSUE THREAD

**Issue by GiovanniF** [Komodo/KomodoEdit#311]
fold "«" SSL_OPERATOR +
fold "»" SSL_OPERATOR -

does not work.

**Comment by Naatan** [comment]
Could you please provide more detailed reproduction steps.

**Comment by GiovanniF** [comment]
I created a language extension using the project template from the menu. Then, in the mylanguage-mainlex.udl file I put

fold '{' SSL_OPERATOR +
fold '}' SSL_OPERATOR -

fold '«' SSL_OPERATOR +
fold '»' SSL_OPERATOR -

After building the language extension, installing it and restarting the editor, when I open a file with the extension I specified for "mylanguage" I can fold text using the braces but not using '«' and '»'.
I also tried 

fold "\xab" SSL_OPERATOR +
fold "\xbb" SSL_OPERATOR -

which contain the codes for '«' and '»'. It might be that since the codes for '«' and '»' are greater than 128 they get special treatment.

Thank you.
